tsconfig.json

```diff
@@ -2,6 +2,12 @@
     "compilerOptions": {
         "lib": ["es2016", "esnext.asynciterable"],
         "baseUrl": "./",
+        "module": "commonjs",
+        "esModuleInterop": true,
+        "target": "es6",
+        "noImplicitAny": true,
```

Comment:
@Kamieniu I recommend adding this option with types support in a separate PR. With current changes application is not building due to Types errors for both `REST` and `GraphQL` APIs.
![Screenshot 2019-11-17 at 21 59 56](https://user-images.githubusercontent.com/16977412/69013977-f1c8f200-0985-11ea-96f7-277e4ff17476.png)